### ISSUE TITLE
Hyperparameter search could choose better (default) loss function

### ISSUE THREAD

**Issue by ToonTalk** [ecraft2learn/ai#42]
The sum of squared errors learns a continuous target variable well. Cross entropy learns categories (in the target). Binary cross entropy as a special case learns just two. 

Could look at the output shape to guess which one.
